pkg/api/constants.go

```diff
@@ -0,0 +1,6 @@
+package api
+
+const (
+	APIEndpoint     = "https://api.opensauced.pizza"
+	BetaAPIEndpoint = "https://beta.api.opensauced.pizza"
+)
```

Comment:
Looks like we have these constants here: https://github.com/open-sauced/pizza-cli/blob/f39986567f5d4e51b35d18bf4b2c1b81eddd09c0/pkg/constants/api.go#L1-L7

Can we use those ones?